policy/modules/session/wayland.if

```diff
@@ -0,0 +1,79 @@
+## <summary>Policy for wayland desktops.</summary>
+
+#########################################
+## <summary>
+##	Associate the specified domain with the
+##	Wayland compositor attribute.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
```

Comment:
This doesn't seem to fit `wayland_compositor_tmpfs_type`.

Reply:
Should be resolved.

Reply:
Perhaps I need to look more at wayland internals... is there a process to handle tmpfs stuff? The attribute naming seems like it should be a file type.

Reply:
In that case, the doc is wrong (parameter name and summary), as the parameter is to be used as a file type, not a domain.